conformance_tests/sysman/test_sysman_temperature/src/test_sysman_temperature.cpp

```diff
@@ -25,6 +25,9 @@ class TempModuleTest : public lzt::SysmanCtsClass {};
 #define TEMPERATURE_TEST TempModuleTest
 #endif // USE_ZESINIT
 
+constexpr uint32_t invalid_max_temperature = 125;
```

Comment:
These variables should be of double type, as get_temp_state() method returns double value.

Reply:
Ok will change the type to double. Thank you for pointing it out.